### PR TITLE
Update Rocket to 0.4

### LINKED ISSUE #546: upgrade to rocket 0.4

### LINKED ISSUE #619: CI failure - "rocket_codegen" could not compile
# Content Bug Report
CI failed for the last 3 PR while compiling the same dependency "rocket_codegen".
The rocket framework was recently updated to v0.4 and there are some changes regarding "rocket_codegen" that could have broken the pipeline.

## Page(s) Affected
CI affected.

ci_failure

## What needs to be fixed?
The project's dependencies.

## Suggested Improvement
Follow the steps from rocket's news page 
https://rocket.rs/v0.4/news/2018-12-08-version-0.4/

rocket_codegen

### PR DESCRIPTION
**Pull request by nrc**
Fixes #546
Fixes #619

r? @ashleygwilliams

### CODE REVIEW

File: templates/governance/group.hbs [hunk 1]
```diff
@@ -2,25 +2,25 @@
 
 <header class="mt3 mt2-ns mb4 mb5-ns tc tl-ns">
   <div class="w-100 mw-none ph3 mw8-m mw9-l center f3">
-    <h1>{{ data.info[0].name }}</h1>
+    <h1>{{ data.info.[0].name }}</h1>
```
Comment (ashleygwilliams) [inline templates/governance/group.hbs]: did the data structure change? i'm curious about this

File: src/main.rs [hunk 2]
```diff
@@ -159,13 +158,13 @@ fn team(t: String, subject: String) -> Template {
...
 
-fn get_type_from_string(s: &str) -> Result<GroupType, Error> {
+fn get_type_from_string(s: &str) -> Result<GroupType, ()> {
     match s {
         "wgs" => Ok(GroupType::WorkingGroup),
         "teams" => Ok(GroupType::Team),
         "peers" => Ok(GroupType::Peer),
         "shepards" => Ok(GroupType::Shepard),
-        _ => Err(Error::Internal),
+        _ => Err(()),
```
Comment (ashleygwilliams) [inline src/main.rs]: is there a reason we don't want to return an error type here anymore?

Comment (nrc) [reply src/main.rs]: The `Error` enum was removed from Rocket, and given that we don't do anything with the error and it doesn't have any data, I thought using `()` was better than creating our own error type

File: templates/governance/group.hbs (hunk 1 above)
Comment (nrc) [reply templates/governance/group.hbs]: Rocket upgraded from Handlebars 0.28 to 1.1 and that brought a breaking change to how array indexes are parsed (see https://github.com/sunng87/handlebars-rust/issues/246).

**Pull request merged**